contracts/ERC721M.sol

```diff
@@ -491,6 +517,15 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
         emit Withdraw(value);
     }
 
+    /**
+     * @dev Withdraws ERC-20 funds by owner.
+     */
+    function withdrawERC20(address mintCurrency) external onlyOwner {
```

Comment:
do we need this input? for example, we can directly use _mintCurrency

Reply:
Yea, this was while there was still a setter, so I guess it can be removed